src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -0,0 +1,285 @@
+(ns quo2.components.navigation.page-nav.view
+  (:require [quo2.components.avatars.group-avatar.view :as group-avatar]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.dropdowns.dropdown :as dropdown]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.navigation.page-nav.style :as style]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]))
+
+(def ^:private button-type
+  {:white       :grey
+   :neutral-5   :dark-grey
+   :neutral-90  :grey
+   :neutral-95  :dark-grey
+   :neutral-100 :black
+   :photo       :grey
+   :blur        :grey})
+
```

Comment:
more of a point for dialog with Designers since the values here are directly from Figma. these button-type keys ("neutral-5", "neutral-90" etc  would be better if the names were not so coupled to the color they are being used and had some semantic expression such as `photo` and `blur` do.
cc @pedro-et  @cammellos